src/Tribe/Traits/Reflection_Tools.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tribe\Traits;
+
+trait Reflection_Tools {
```

Comment:
I feel like we shouldn't need this, I would love to see us take a more descriptive approach, like mentioned on some of the other comments.